src/main/java/seedu/address/model/person/ClassTiming.java

```diff
@@ -41,20 +50,19 @@ public ClassTiming(String classTiming) {
      * @param classTiming classTiming where day is going to be changed to caps.
      * @return classTiming with the day in caps.
      */
-    public String formatClassTiming(String classTiming) {
+    private String formatClassTiming(String classTiming) {
         String day = classTiming.split(" ")[0].toUpperCase();
         String timing = classTiming.split(" ")[1];
         return day + " " + timing;
     }
 
     /**
-     * Converts the day into the int representation of the day.
+     * Converts Day from the ClassTiming object into the int representation of the day.
      *
-     * @param day String value of the day in class timing.
-     * @return int representatin of the days of the week.
+     * @return int representation of the Day of the ClassTiming object
      */
-    public int replaceDayWithInt(String day) {
-        switch (day) {
+    public int getDayToInt() {
+        switch (day.toUpperCase()) {
```

Comment:
Yeah I think it would be better if we use an enum for this (if we have time)

Reply:
yeah day we can change to enum in the future, now not necessary yet especially since we are rushing to push out other important features